src/Pachyderm/Orm/Model.php

```diff
@@ -348,11 +348,18 @@ public static function pagination(array $params): SQLBuilder
         unset($params['order']);
 
         if (!empty($params['filter'])) {
-            $query = new QueryBuilder($params['filter']);
+            $conditions = json_decode($params['filter']);
```

Comment:
Why is it decoding json?

Reply:
the filter passed here is in json format, here it parses it into array

Reply:
I can understand the code, I cannot understand the reason.